.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run lint
```

Comment:
We have lint staged installed but we are not using it. It would be better to lint stage and also to format files using prettier and **fix** the issues with lint before displaying errors in pre-vcommit